notebooks/1_table_oriented.ipynb

```diff
@@ -0,0 +1,484 @@
+{
```

Comment:
<p>I'd remove this comment as well - I don't think we care to draw that comparison this early on</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stijnvanhoey/pandas-getting-started-tutorials/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>If you decide to keep this language, I would change "When you are familiar" to "If you are familiar".</p><p> </p><p>Also, "base" should be "based".</p>

Reply:
<p>I left the note there for the moment. I do think Python users will _like_ the link with something they already know.</p>